# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1502380405 14400
#      Thu Aug 10 11:53:25 2017 -0400
# Node ID 3487b4a1e4d04254a8bfd907696592e8b19df423
# Parent  8f2d51b6235f5a34eb5beed622f596af29333467
* import.m: Escape brace characters in docstring.

diff --git a/scripts/path/import.m b/scripts/path/import.m
--- a/scripts/path/import.m
+++ b/scripts/path/import.m
@@ -34,17 +34,17 @@
 ## less typing by programmers.
 ##
 ## Example
 ##
 ## @example
 ## @group
 ## import containers.Map;
 ##
-## m = Map ({"A", "B"}, {[1], [2]});
+## m = Map (@{"A", "B"@}, @{[1], [2]@});
 ## @end group
 ## @end example
 ##
 ## When called with no inputs and no outputs @code{import} prints a list of
 ## any import definitions.
 ##
 ## When called with no inputs and one output, a cell array of strings
 ## @var{list} is returned with any import definitions.
